ompi/mca/coll/base/coll_tags.h

```diff
@@ -45,6 +45,7 @@
 #define MCA_COLL_BASE_TAG_NONBLOCKING_END ((-1 * INT_MAX/2) + 1)
 #define MCA_COLL_BASE_TAG_NEIGHBOR_BASE  (MCA_COLL_BASE_TAG_NONBLOCKING_END - 1)
 #define MCA_COLL_BASE_TAG_NEIGHBOR_END   (MCA_COLL_BASE_TAG_NEIGHBOR_BASE - 1024)
-#define MCA_COLL_BASE_TAG_HCOLL_BASE (-1 * INT_MAX/2)
+#define MCA_COLL_BASE_TAG_UCC (MCA_COLL_BASE_TAG_NEIGHBOR_END - 1)
+#define MCA_COLL_BASE_TAG_HCOLL_BASE (MCA_COLL_BASE_TAG_UCC - 1)
```

Comment:
Can someone from MLNX confirm that this won't cause any problems?  It looks like it does have the practical effect of changing the base tag for HCOLL.  Not that I think HCOLL and NONBLOCKING each grabbing almost half of the tag space was a reasonable thing to do, but here we are :).

Reply:
Yes, i can confirm, hcoll is good with that change. In fact, before the change there was a collision of tag (-1 * INT_MAX/2) as it was used as both NEIGHBOR_BASE ( -1 * INT_MAX/2 + 1 - 1) and HCOLL_BASE.

Reply:
Thanks.  I think we're good to go once we ship 4.1.3.